code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -25,7 +25,6 @@
 
 	var/sanity_gain = 0.2 //per nutriment
 	var/junk_food = FALSE //if TRUE, sanity gain per nutriment will be zero
-	var/cooked = FALSE
```

Comment:
Need this to be marked as Occulus Edit since it still exists upstream

Reply:
Sanity gain per bite code seems to be still in. Is this intentional?

Reply:
That's intentional, and I thought I didn't have any noncore files. That's something I can fix.

Reply:
Deletion of this line need to be a comment and marked as Occulus Edit